sos/plugins/block.py

```diff
@@ -39,19 +38,13 @@ def setup(self):
             "/sys/block/*/queue/scheduler"
         ])
 
-        if os.path.isdir("/sys/block"):
-            for disk in os.listdir("/sys/block"):
-                if disk.startswith("ram"):
-                    continue
-                disk_path = os.path.join('/dev/', disk)
-                queue_path = os.path.join('/sys/block/', disk, 'queue')
-                self.add_udev_info(disk_path)
-                self.add_udev_info(disk_path, attrs=True)
-                self.add_udev_info(queue_path, attrs=True)
```

Comment:
I think this command is not carried forward. At least when running sosreport without and with this PR and comparing collected files, these four are missing on my system:

    sos_commands/block/udevadm_info_-a_.sys.block.dm-0.queue
    sos_commands/block/udevadm_info_-a_.sys.block.dm-1.queue
    sos_commands/block/udevadm_info_-a_.sys.block.sda.queue
    sos_commands/block/udevadm_info_-a_.sys.block.sr0.queue

(not sure if that data is important to collect or not, just spotting difference / potential regression)

Reply:
I had not moved those forward because on RHEL 8 and Fedora 28+, it does not appear to generate useful information:

RHEL:
```
# udevadm info -a /sys/block/sda/queue/
Unknown device, absolute path in /dev/ or /sys expected.
# udevadm info -a /sys/block/sda/queue
Unknown device, absolute path in /dev/ or /sys expected.
# udevadm info -a /sys/block/dm-1/queue/
Unknown device, absolute path in /dev/ or /sys expected.
```

Fedora:
```
Unknown device "/sys/block/dm-0/queue": No such device
Unknown device "/sys/block/nvme0n1/queue": No such device
Unknown device "/sys/block/sda/queue": No such device
```


But from the PR that added it, it says "very important". @jtpittman195  can you shed any light on this? Is ^ still important or do we need to change how we're calling it?

Reply:
Jake, sorry for the delay.  You're right, it fails on rhel8 and fedora due to a change in systemd.  Since sys/block/sdc/queue is not a device, udevadm rejects it.  The values it grabbed were ones that we used often with tuning and alignment cases, so if you're able to grab them differently via a sysfs copy or the like, that would be immensely helpful to us.  Thanks again.